arch/M680X/M680XDisassembler.c

```diff
@@ -1114,9 +1113,8 @@ static void reg_bits_hdlr(MCInst *MI, m680x_info *info, uint16_t *address)
 		break;
 
 	default:
-		fprintf(stderr, "Internal error: Unexpected operand0 register "
-			"%d\n", op0->reg);
-		abort();
+		//assert(0 && "Unexpected operand0 register");
```

Comment:
Why do we want to commit commented out code?

Reply:
As I already have stated in the commit message "The commented out assert's help the maintainer when e.g. adding new M680X cpu types.".
This seems to be common sense within this project as You can see with "grep -rn assert".
As You also will see is that RISCV still has assert() not commented out :-)

Reply:
Instead of commenting the code out, how about we add a conditional macro to `cs_priv.h` such as:

~~~c
#ifdef CAPSTONE_DEBUG
#define CS_ASSERT(expr) assert(expr)
#else
#define CS_ASSERT(expr) 0
#endif
~~~

We can use `CS_ASSERT` throughout capstone (including other architectures). If a dev wants to enable the asserts, then she can just add `-DCAPSTONE_DEBUG` to the compiler invocation (or whatever is appropriate for her compiler).

That way, devs would no longer need to manually enable/disable each assert.